changelog/seqera-enterprise/v25.1.mdx

```diff
@@ -0,0 +1,120 @@
+- **Custom Studio session lifespans**: This new feature allows you to set a maximum lifespan for a session, after which time the session is stopped automatically and a checkpoint created, or the session can be extended on-demand.
+- **Studios resource labels support**: Workspace maintainer role and above users can manage the resource labels inherited from the compute environment and attached to the Studio. Resource labels attached to each Studio don’t affect the default resource labels associated with the compute environment.
+
+### Fusion
+
+- AWS Batch compute environments default to Amazon Linux 2023 AMI when Fusion v2 is enabled.
+- Add support for Fusion licensing.
+
+### General
+
+- Updated list of EC2 families with NVMe disk available 
+- Audit log update: Pipeline edit events are now logged.
+- Switch AWS Batch compute environment dependencies to AWS SDK v2.
+- Switch Compute dependencies to AWS SDK v2.
+- Support image upload in the pipeline import and pipeline update forms, to be used as pipeline logos.
+- New dynamic page title for easy bookmarking.
+- Added `totalProcesses` to workflow progress responses.
+- Implement collapsible view for JSON workflow parameters tab and add **View as YAML** option.
+- Update the pipeline name regular expression to allow pipeline names containing dots ('.').
+- Extend zombie jobs detection on all workflows reaching one of the terminal states (`SUCCEEDED`, `FAILED`, `CANCELLED`).
+- Allow Nextflow configuration parameters with embedded references to other parameters to be shown verbatim.
+- Improved error messaging when pipeline info can't be fetched indicates whether the issue is due to a missing resource, or failed authentication due to expired credentials.
+- Azure jobs are now automatically terminated after all tasks are complete.
+- Send only added/updated run parameters when launching a pipeline. This includes all defaults and parameters passed during the launch.
+- Allow users to remove organization logo.
+- New workflow job monitoring collects and publishes Platform metrics.
+- Notify the user when secrets cleanup fails.
+- Dashboard page: Add a date filter. 
+- Bump nextflow to version 24.10.3
+- Upgrade to Angular 18
+
+## Bug fixes
+
+- Studios
+  - Fixed a bug where resource labels were erroneously editable for the maintain role when starting an existing Studio.
+  - Fixed an issue to ensure that the Maintain role is not able to change resource labels inherited from the selected compute environment when adding/starting a Studio.
+  - Fixed an issue with searches for names containing special SQL wildcard characters (_, %).
+- Prohibit duplicate Git credentials and tie-break on `lastUpdated` for existing duplicates. Going forward, preference is given to the credentials updated last in the situation when there are multiple candidates. Additionally a duplicate check is added when creating new credentials, to avoid new duplicate cases.
```

Comment:
```suggestion
- Prohibit duplicate Git credentials and tie-break on `lastUpdated` for existing duplicates. Going forward, preference is given to the credentials updated last in the situation when there are multiple candidates. Additionally, a duplicate check is added when creating new credentials to avoid new duplicate cases.
```